docs/help/policies/versioning.mdx

```diff
@@ -0,0 +1,96 @@
+   through minor or patch versions.
+2. **Security updates**: Updates are provided via patches to address known security vulnerabilities.
+3. **Technical support**: Assistance is provided for issues or questions about individual components.
+
+Support is available for the latest minor version of the previous major version of each component, including critical
+security updates, for up to one year after the release of the current major version of that component.
+
+You can find the latest versions and changelog entries for each component on the
+[Hasura Changelog](https://hasura.io/changelog).
+
+## Reporting bugs and requesting fixes
+
+Hasura is committed to providing enterprise-grade reliability and support to ensure your business operates seamlessly.
+For bug reporting and resolution, the following process ensures efficiency and transparency:
+
+1. **Reporting Bugs**:
+
+   - Bugs can be reported through the [Hasura Support Portal](https://support.hasura.io/hc/en-us/requests/new).
+   - Customers are encouraged to provide detailed descriptions, logs, and replication steps to help expedite resolution.
+
+2. **Bug Triage and Prioritization**:
+
+   - Reported issues are triaged based on severity and impact, prioritizing critical bugs that affect system security,
+     availability, or functionality.
+   - Issues impacting production environments are escalated for immediate attention.
+
+3. **Bug Fix Releases**:
+   - Fixes for critical issues are rolled out as patch updates. Non-critical issues are included in the subsequent minor
+     or major releases.
+   - Customers are notified through release notes, the [Hasura Changelog](https://hasura.io/changelog), and direct
+     communication for mission-critical updates.
+
+## Security updates and communication
+
+Security is a top priority for Hasura. To maintain the integrity of your systems, we adhere to the following practices:
+
+1. **Proactive Security Patching**:
+
+   - Vulnerabilities are addressed promptly with targeted patch updates.
+   - Security patches are backward-compatible and applied to supported versions.
+
+2. **Customer Communication**:
+
+   - Security updates are communicated directly to enterprise customers through our Customer Success team and email
+     notifications.
+   - Detailed documentation and guidance are provided to ensure smooth implementation of critical updates.
+
+## Rolling out updates
+
+Hasura ensures a seamless update experience for enterprise customers by following these principles:
+
+1. **Pre-Release Testing**:
+
+   - Updates are rigorously tested across supported versions and environments to minimize disruption.
+
+2. **Documentation and Support**:
+   - Comprehensive release notes and upgrade guides accompany every update.
+   - Our support team is available to assist with planning and executing updates, ensuring minimal impact on operations.
```

Comment:
@robertjdominguez we mention engine, connectors, and CLI, but only touch lightly on engine using calver. So no mention of how connectors/CLI are versioned. Also as a user if components are independently versioned how do the independent versions matter to me. Ie how do I as a user know which versions or which components to use and/or when and what to upgrade. I feel like we need more information and guidance here for users on versioning and how it all fits together for them.

Reply:
Got it 👍 

I'll make some tweaks on Monday and ping you via Slack.